samples/azure-quantum/hello-world/HW-rigetti-qiskit.ipynb

```diff
@@ -0,0 +1,380 @@
+      "id": "1b094a8a-76f0-4a49-a0b6-3736f73c18a8",
+      "metadata": {},
+      "source": [
+        "### 2. Build the quantum program\n",
+        "\n",
+        "Let's create a simple Qiskit circuit to run."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 45,
+      "id": "7b42a2c4-cf76-4678-bdce-54ba6d6679e6",
+      "metadata": {},
+      "outputs": [
+        {
+          "data": {
+            "text/html": [
+              "<pre style=\"word-wrap: normal;white-space: pre;background: #fff0;line-height: 1.1;font-family: &quot;Courier New&quot;,Courier,monospace\">     ┌───┐┌─┐\n",
+              "  q: ┤ H ├┤M├\n",
+              "     └───┘└╥┘\n",
+              "c: 1/══════╩═\n",
+              "           0 </pre>"
+            ],
+            "text/plain": [
+              "     ┌───┐┌─┐\n",
+              "  q: ┤ H ├┤M├\n",
+              "     └───┘└╥┘\n",
+              "c: 1/══════╩═\n",
+              "           0 "
+            ]
+          },
+          "execution_count": 45,
+          "metadata": {},
+          "output_type": "execute_result"
+        }
+      ],
+      "source": [
+        "# Create a quantum circuit acting on a single qubit\n",
+        "circuit = QuantumCircuit(1,1)\n",
+        "circuit.name = \"Single qubit random\"\n",
+        "circuit.h(0)\n",
+        "circuit.measure(0, 0)\n",
+        "\n",
+        "# Print out the circuit\n",
+        "circuit.draw()"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "id": "23f20a40-c972-470d-9c0e-5cd6c1c2eb38",
+      "metadata": {
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        }
+      },
+      "source": [
+        "The circuit you built is a simple quantum random bit generator. With Rigetti's simulator, we will be able to simulate the probability of measuring a `1` or `0`."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "id": "034aec28-a624-4d9a-9899-67b69543d82d",
+      "metadata": {},
+      "source": [
+        "### 3. Submit the quantum program to Rigetti"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 46,
+      "id": "486ab624-bfde-4ae5-971d-02727620938f",
+      "metadata": {
+        "jupyter": {
+          "outputs_hidden": false,
+          "source_hidden": false
+        },
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        }
+      },
+      "outputs": [
+        {
+          "name": "stdout",
+          "output_type": "stream",
+          "text": [
+            "Job id: 444cc976-ec31-11ec-95ba-00155d507810\n"
+          ]
+        }
+      ],
+      "source": [
+        "from azure.quantum.target.rigetti import RigettiTarget\n",
+        "\n",
+        "# Create an object that represents Rigetti's simulator target, \"rigetti.sim.qvm\" using the packaged constant.\n",
+        "#   Note that any target you have enabled in this workspace can\n",
```

Comment:
I believe this is only true for QC targets; I'm not sure what it would mean to submit a Qiskit object to a QIO target.
```suggestion
        "#   Note that any target you have enabled in this workspace can\n",
```